app/gui/src/controller/ide/plain.rs

```diff
@@ -83,8 +83,8 @@ impl Handle {
 }
 
 impl controller::ide::API for Handle {
-    fn current_project(&self) -> model::Project {
-        self.project.clone_ref()
+    fn current_project(&self) -> Option<model::Project> {
+        Some(self.project.clone_ref())
```

Comment:
I don't understand this implementation. It always returns `Some`, so what is the reason of it returning `Option` at all?